app/templates/index.html

```diff
@@ -56,24 +56,24 @@
 						<div class="menuButtonContainter">
 							<div ng-switch="temperatureStatus">
 				    			<button ng-click="buttonStatus(temperatureStatus, 'learn')" class="btn btn-block-green" ng-switch-when="0"><i class="fas fa-check-circle"></i><br>Temperature<br/>{a temperature a} {a temperatureUnits a}</button>
-				    			<button ng-click="buttonStatus(temperatureStatus, 'learn')" class="btn btn-block-yellow" ng-switch-when="1"><i class="fas fa-exclamation-triangle"></i><br>Temperature<br/>{a temperature a} {a temperatureUnits a}</button>
-				    			<button ng-click="buttonStatus(temperatureStatus, 'learn')" class="btn btn-block-red" ng-switch-when="2"><i class="fas fa-exclamation-circle"></i><br>Temperature<br/>{a temperature a} {a temperatureUnits a}</button>
+				    			<button ng-click="buttonStatus(temperatureStatus, 'http://11.11.11.11/digital_library/PlantHealth/Crops/Coconut')" class="btn btn-block-yellow" ng-switch-when="1"><i class="fas fa-exclamation-triangle"></i><br>Temperature<br/>{a temperature a} {a temperatureUnits a}</button>
```

Comment:
You need to have .html at the end of the link. It should be http://11.11.11.11/digital_library/PlantHealth/Crops/Coconut.html